root/static/scripts/guess-case/MB/GuessCase/Main.js

```diff
@@ -119,9 +119,10 @@ const flags = require('../../flags');
         sortname: guess("Work", "guessSortName")
     };
 
-    // Series doesn't have it's own handler, and just uses the work handler
+    // Series and Event doesn't have it's own handler, and just uses the work handler
```

Comment:
You added another entity type, but the rest of the sentence is still in singular.

Also, you might want to wrap the line. Not sure how long it is now or if MBS has a suggested/hard line length limit. Either way, this is easy to wrap, so let's do it. :)